app-admin/u-boot-menu/autobuild/postinst

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+
+if [ "$1" = "triggered" ]
```

Comment:
I'm confused as all branches seem to lead to the same thing (whether when triggered or when configuring, or when `/boot/extlinux` does not exist.

Reply:
I referred to the relevant triggers in `dracut-ng` from ABBS and streamlined the content.